index.js

```diff
@@ -3241,6 +3264,83 @@
     impl: Z.size
   };
 
+  //# all :: Foldable f => (a -> Boolean) -> f a -> Boolean
+  //.
+  //. Returns `true` [iff][] all the elements of the structure satisfy the
+  //. predicate.
+  //.
+  //. See also [`any`](#any) and [`none`](#none).
+  //.
+  //. ```javascript
+  //. > S.all (S.odd) ([])
+  //. true
+  //.
+  //. > S.all (S.odd) ([1, 3, 5])
+  //. true
+  //.
+  //. > S.all (S.odd) ([1, 2, 3])
+  //. false
+  //. ```
+  _.all = {
+    consts: {f: [Z.Foldable]},
+    types: [$.Predicate (a), f (a), $.Boolean],
+    impl: curry2 (Z.all)
+  };
+
+  //# any :: Foldable f => (a -> Boolean) -> f a -> Boolean
+  //.
+  //. Returns `true` [iff][] any element of the structure satisfies the
+  //. predicate.
+  //.
+  //. See also [`all`](#all) and [`none`](#none).
+  //.
+  //. ```javascript
+  //. > S.any (S.odd) ([])
+  //. false
+  //.
+  //. > S.any (S.odd) ([2, 4, 6])
+  //. false
+  //.
+  //. > S.any (S.odd) ([1, 2, 3])
+  //. true
+  //. ```
+  _.any = {
+    consts: {f: [Z.Foldable]},
+    types: [$.Predicate (a), f (a), $.Boolean],
+    impl: curry2 (Z.any)
+  };
+
+  //# none :: Foldable f => (a -> Boolean) -> f a -> Boolean
+  //.
+  //. Returns `true` [iff][] none of the elements of the structure satisfies
```

Comment:
```suggestion
  //. Returns `true` [iff][] none of the elements of the structure satisfy
```

I believe you've made a copy-paste typo here.

Reply:
A thorough review as always, Aldwin!

You have highlighted a [contentious facet of English grammar][1]. I favour _**none** of the … **satisfies**_ as it corresponds to _**not one** of the … **satisfies**_. I consider both options correct, but I prefer _satisfies_ in this sentence.


[1]: https://english.stackexchange.com/q/1425

Reply:
I had no idea. Using _none_ as plural seems so foreign to me that I just assumed it was incorrect. I would prefer _satisfy_, but I guess we're just talking about personal preference at this point.

Reply:
> Using _none_ as plural seems so foreign to me that I just assumed it was incorrect.

To be clear, the usage you find foreign is the singular:

  - _**one** of these pens **is** red_
  - _**not one** of these pens **is** red_
  - _**none** of these pens **is** red_

The usage you prefer is the plural:

  - _these pens **are** red_
  - _**none** of these pens **are** red_

> I guess we're just talking about personal preference at this point.

The spelling and grammar of the documentation reflect my reproducibly inconsistent preferences. :)

Reply:
> To be clear, the usage you find foreign is the singular

Ah, yes. My bad.